From cbb6e999265431ee4be9e3952864bb07f1a40c77 Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Wed, 24 Sep 2014 17:42:30 -0400
Subject: [redhat] configs: Disable Amiga partition table support for power LE

Message-id: <20140924174230.5118.76690.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 96706
O-Subject: [PATCH RHEL7.1 BZ1145615 2/2] redhat Kconfig: Disable Amiga partition table support for power LE
Bugzilla: 1145615
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1145615

Description:
------------
Disable Amiga partition table support for power LE.

RHEL Version Found:
-------------------
7.1

Upstream Status:
----------------
n/a

----------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/powerpc64le/CONFIG_AMIGA_PARTITION b/redhat/configs/generic/powerpc64le/CONFIG_AMIGA_PARTITION
new file mode 100644
index 0000000..300a97b
--- /dev/null
+++ b/redhat/configs/generic/powerpc64le/CONFIG_AMIGA_PARTITION
@@ -0,0 +1 @@
+# CONFIG_AMIGA_PARTITION is not set
-- 
1.7.1